editor/import/3d/post_import_plugin_skeleton_rest_fixer.cpp

```diff
@@ -204,7 +204,7 @@ void PostImportPluginSkeletonRestFixer::internal_process(InternalImportCategory
 								} else {
 									for (int j = 0; j < key_len; j++) {
 										Basis sc = Basis().scaled(static_cast<Vector3>(anim->track_get_key_value(i, j)));
-										anim->track_set_key_value(i, j, (global_transform.basis * sc).get_scale());
+										anim->track_set_key_value(i, j, (global_transform.orthonormalized().basis * sc).get_scale());
```

Comment:
Do we need to orthonormalize the transform or just the basis? Like so:
```suggestion
										anim->track_set_key_value(i, j, (global_transform.basis.orthonormalized() * sc).get_scale());
```

Reply:
To be honest, I am not sure. The point is that this calculation should be identical to line 136, where they also wrote it this way.

Reply:
Looking at the code in `Transform3D` it's identical, wasn't sure if there was some unnecessary computations done in the `Transform3D` method so it's equivalent